### PR TITLE
[Upcoming] Release 1.5.0 

### PR DESCRIPTION
**Pull request by skoudoro**
Preparation for 1.5.0 release, targeting ~~November 7th-8th~~  March 1st. 

# Desired PR (need review / feedback / update)

There is room for the following PR to be finished and merge

- [x] #2533  (Submitter: @skoudoro)
- [x] #2512 (Submitter: @samcoveney)
- [x] #2442 (Submitter: @skoudoro)
- [x] #2423 (Submitter: @kenjimarshall)
- [x] #2422 (Submitter: @nasimanousheh)
- [x] #2420 (Submitter: @nasimanousheh)
- [ ] #2369 (Submitter: @skoudoro)
- [x] #2342 (Submitter:@kerkelae)
- [x] #2461
- [x] #2463 

Most of them can be done on time, I think :smile:

Please comment to flag any other issues or PR that should be addressed for this release

# Release checklist
- [x] add release_notes 1.5.0
- [ ] Update changelog and api_changes
- [x] Update .mailmap
- [ ] Set release number in info.py
- [x] Check the copyright years in doc/conf.py and LICENSE
- [x] Check documentation
- [x] Check Tutorial
- [x] Check unit test and USE-PRE
- [x] Update index.rst
- [x] Update Highligth.rst, old_news.rst, stateofheart.rst
- [x] Update developers.rst list
- [ ] Update website
- [ ] Update deprecation cycle remove long deprecation

### CODE REVIEW

**Comment by skoudoro** [comment]
yes, added. #2463  also

**Comment by skoudoro** [comment]
I think we are ready for the Release.  If  I find time, I will look https://github.com/dipy/dipy/pull/2369 but it should not block the release.

Also, we need to check documentation generation and if there is no surprise with the wheels.

**Comment by skoudoro** [comment]
As you can see [here](https://github.com/MacPython/dipy-wheels/actions/runs/1925624178 ), the wheels for manyLinux python 3.10 are failing. If someone has any idea on how to fix that. I will appreciate it a lot the help!

@arokem? @matthew-brett @Garyfallidis?

**Comment by arokem** [comment]
Sorry: this is above my head.

**Comment by larsoner** [comment]
The error is `ls: /opt/python/cp310-cp310/bin: No such file or directory`. Looks like you're using `pypa/manylinux1_x86_64`. Maybe you need to upgrade to manylinux2010 or later to get 3.10 support? Looks like NumPy and SciPy intentionally removed references to manylinux1 in favor of manylinux:

https://github.com/MacPython/numpy-wheels

I would look to see how they have updated to get 3.10 working...

**Comment by skoudoro** [comment]
Thank you for looking into this @larsoner.

>  Looks like you're using pypa/manylinux1_x86_64

yes, we are using this in one matrix.

> Maybe you need to upgrade to manylinux2010

Another matrix use this one as you can see [here]() but it fails also with the following error:

```python
ERROR: Exception:
Traceback (most recent call last):
  File "/venv/lib/python3.10/site-packages/pip/_internal/cli/base_command.py", line 173, in _main
    status = self.run(options, args)
  File "/venv/lib/python3.10/site-packages/pip/_internal/cli/req_command.py", line 203, in wrapper
    return func(self, options, args)
  File "/venv/lib/python3.10/site-packages/pip/_internal/commands/install.py", line 390, in run
    installed = install_given_reqs(
  File "/venv/lib/python3.10/site-packages/pip/_internal/req/__init__.py", line 73, in install_given_reqs
    requirement.install(
  File "/venv/lib/python3.10/site-packages/pip/_internal/req/req_install.py", line 723, in install
    scheme = get_scheme(
  File "/venv/lib/python3.10/site-packages/pip/_internal/locations/__init__.py", line 155, in get_scheme
    new = _sysconfig.get_scheme(
  File "/venv/lib/python3.10/site-packages/pip/_internal/locations/_sysconfig.py", line 131, in get_scheme
    scheme_name = _infer_prefix()
  File "/venv/lib/python3.10/site-packages/pip/_internal/locations/_sysconfig.py", line 45, in _infer_prefix
    return sysconfig.get_preferred_scheme("prefix")  # type: ignore
AttributeError: module 'sysconfig' has no attribute 'get_preferred_scheme'
```
I suppose, the VM needs a more recent version of pip. I just do not know how to update it.

> or later to get 3.10 support?

I will experiment with a newer one.

Also, it seems that Numpy migrates to [cibuildwheel](https://github.com/pypa/cibuildwheel) for building Linux and macOS wheels as you can see [here](https://github.com/numpy/numpy/pull/20102)

We might do the same move. I will experiment that this morning.

**Comment by skoudoro** [comment]
ok, manylinux1 has been dropped for python3.10, makes sense now. More info on https://github.com/pypa/manylinux/issues/994

**Comment by skoudoro** [comment]
ok, all good with python3.10 and manylinux2014.  I will explore [cibuildwheel](https://github.com/pypa/cibuildwheel) another day. 

Finalizing doc generation, checking tutorials and examples.

File: Changelog [hunk 1]
```diff
@@ -24,6 +24,22 @@ Dipy
 
 The code found in Dipy was created by the people found in the AUTHOR file.
 
+* 1.5.0 (Thursday, March 11, 2022)
```
Comment (larsoner) [inline Changelog:27]:
```suggestion
* 1.5.0 (Friday, March 11, 2022)
```

**Comment by arokem** [comment]
LGTM! Ready to go, I believe. 

🚀 ❗ ❗ ❗ ⚡

File: Changelog (hunk 1 above)
Comment (skoudoro) [reply Changelog:27]: good catch

**Comment by Garyfallidis** [comment]
Release time! Let's GO! :100:

**Pull request merged**